### ISSUE TITLE
encoding issue when translated to RO

### ISSUE THREAD

**Issue by samcux** [alos-source/psTranslateExcel#1]
here an exampe:

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 12">

</head>

<body link=blue vlink=purple>

invalid   device adress | AdresÄ dispozitiv invalidÄ
-- | --

</body>

</html>

any ideas how this can be fixed?

**Comment by alos-source** [comment]
You are right, there seems to be a general problem with the encoding of non ascii chars. 
I also could verify with my texts.
I've tried some things to get the encoding right today, but no improvement in the behavior so far.
